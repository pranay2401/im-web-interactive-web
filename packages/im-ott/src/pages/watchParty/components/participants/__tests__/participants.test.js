import React from "react";
import { cleanup } from "@testing-library/react";
import Participants from "../Participants";
import renderer from "react-test-renderer";

afterEach(cleanup);

jest.mock("react-router-dom", () => ({
  Link: "Link",
  useLocation: jest.fn().mockReturnValue({
    pathname: "/search",
    search: "",
    hash: "",
    state: null,
    key: "5nvxpbdafa",
  }),
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

it("Participants should render correctly", () => {
  const tree = renderer.create(<Participants />).toJSON();
  expect(tree).toMatchSnapshot();
});
