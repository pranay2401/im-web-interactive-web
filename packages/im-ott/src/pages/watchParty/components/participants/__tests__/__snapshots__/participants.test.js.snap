// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participants should render correctly 1`] = `
<div
  className="container"
>
  0
  1
  2
  3
  4
  5
  6
  7
  8
  9
  10
  11
  12
  13
  14
  15
</div>
`;
